getting_started/eyelink.md

```diff
@@ -19,23 +19,23 @@ It is possible to use the eye-tracker in combination with a behavioral task that
 
 ### Converting the EDF file to an ASC file
 
-The Eyelink 1000 output is an _.EDF file (eyelink data file). There are several ways to read (directly or indirectly) this data into MATLAB. To use FieldTrip for data analysis, you have to convert the EDF file to an ASCII text format using the Eyelink executable EDF2ASC.EXE. You need to run the EDF2ASC.EXE that can be downloaded [here](https://www.sr-support.com/showthread.php?17-EDF2ASC-Conversion-Utility) under MSDOS on the Eyelink computer and find your _.EDF recording as follow
```

Comment:
This could be updated? AFAIK, at the Donders this is now done on the Stimulus PC, which is way more convenient.